pysat/tests/classes/cls_instrument_access.py

```diff
@@ -0,0 +1,1003 @@
+"""Tests for data access and related functions in the pysat Instrument object.
+
+Includes:
+* data loading
+* equality method comparisons
+* index tests
+* setter and getter functions
+* concat
+* empty data flags
+* variable renaming
+
+Note
+----
+Base class stored here, but tests inherited by test_instrument.py
+
+"""
+
+import datetime as dt
+from importlib import reload
+import numpy as np
```

Comment:
We should probably come up with a standard and sweep the code.  I think @aburrell and I discussed at one point keeping `numpy` as part of the core packages.

Reply:
Yeah, I think this would be good to establish during a meeting, add a standard to the development guidelines, and do a sweep.  Not part of this pull, though 🐦